includes/data-events/class-utils.php

```diff
@@ -63,6 +63,8 @@ public static function get_order_data( $order_id, $process_donations_only = fals
 				'product_id' => $product_id,
 				'client_id'  => $order->get_meta( NEWSPACK_CLIENT_ID_COOKIE_NAME ),
 			],
+			'user_first_name' => $subscription->get_billing_first_name(),
+			'user_last_name'  => $subscription->get_billing_last_name(),
```

Comment:
Should be `$order`, there's no `$subscription` defined in this function.

```suggestion
			'user_first_name' => $order->get_billing_first_name(),
			'user_last_name'  => $order->get_billing_last_name(),
```

Reply:
oops. fixed